scudcloud-1.0/lib/wrapper.py

```diff
@@ -106,6 +106,12 @@ def _linkClicked(self, qUrl):
         else:
             self.systemOpen(url)
 
+    def gotoNextOpenChannelOrIM(self, unreads_only=False, backwards=False):
+        return self.page().currentFrame().evaluateJavaScript(
+            'TS.client.ui.gotoNextOpenChannelOrIM(%s, %s);' % (
+                str(bool(unreads_only)).lower(),
+                str(bool(backwards)).lower()))
+
```

Comment:
Ok, added- although it still doesn't use self.call(), since that doesn't allow for multiple arguments right now. Is that better?